src/controllers/user.controller.js

```diff
@@ -26,4 +26,22 @@ export default class UserController {
       });
     }
   }
+
+  async userLogin(req, res, next) {
+    try {
+      const user = await this.userService.userLogin(req.body.email, res);
+      const validation = await comparePassword(req.body.password, user.password)
+      if(validation) {
+        const token = await generateToken({email: user.email}, "1d")
+        res.status(201).header("authenticate", token).json({message: "Logged in successfully", token: token})
+      }
+      else {
+        res.status(401).send("Invalid password")
+      }
+    } catch (error) {
+      console.log(error)
+      res.status(404).send({error: error})
```

Comment:
- Remove the console and add provide a meaningful message to the user
`res.status(500).send({error: error, message: ""})`
- change status code